nucypher/network/nodes.py

```diff
@@ -1347,30 +1322,40 @@ def node_details(node):
         except AttributeError:
             last_seen = str(node.last_seen)  # In case it's the constant NEVER_SEEN
 
-        fleet_icon = node.fleet_state_nickname
-        if fleet_icon is UNKNOWN_FLEET_STATE:
-            fleet_icon = "?"  # TODO  NRN, MN
-        else:
-            fleet_icon = fleet_icon.icon
-
         payload = {"icon_details": node.nickname.payload(),
                    "rest_url": node.rest_url(),
                    "nickname": str(node.nickname),
                    "worker_address": node.worker_address,
                    "staker_address": node.checksum_address,
                    "timestamp": node.timestamp.iso8601(),
                    "last_seen": last_seen,
-                   "fleet_state": node.fleet_state_checksum or 'unknown',
-                   "fleet_state_icon": fleet_icon,
                    "domain": node.serving_domain,
                    'version': nucypher.__version__}
         return payload
 
+    def _known_nodes_details(self) -> dict:
```

Comment:
General commentary - Let;s document where these metadata / description methods are being used, for an example: If a method is used for HTTP rendering, CLI, etc.  Consider reducing the number of these descriptor methods where possible.

Reply:
Now this info is used both for JSON and HTML status responses.